### PR TITLE
Enable all available layers

### PR DESCRIPTION
**Pull request by urish**
PR #28 with requested changes

### CODE REVIEW

File: inkscape/svg2shenzhen/export.py [hunk 1]
```diff
@@ -457,6 +458,13 @@ def processExportLayer(self):
                     processes.append(p)
                 for p in processes:
                     p.wait()
+
+                    # Open generated file to replace the wrong layer string
+                    mod = open(layer_dest_kicad_path, "r").read()
+                    mod = mod.replace('F.Cu', layer_label)
+                    open(layer_dest_kicad_path, "w").write(mod)
+
+
```
Comment (kasbah) [inline inkscape/svg2shenzhen/export.py]: Can we not fix this in bitmap2component instead?

Comment (urish) [reply inkscape/svg2shenzhen/export.py]: I'm not familiar with bitmap2component, but I can try to look into that. Are there any build instructions / pointers that can help here?

**Comment by urish** [comment]
I updated the source code for bitmap2component so that any layer name can be used with it. I will test the new version tomorrow and let you know when I'm done

**Comment by urish** [comment]
Alright, tested and working for my project. I updated `export.py` as well to be backward compatible.

Do you want me to add the updated windows executable of `bitmap2component`  to this PR as well?

**Comment by urish** [comment]
For future reference, this is how I built the the windows executable:

1. Downloaded [this msysgw distribution](https://nuwen.net/mingw.html) which includes the Boost library. Specifically, I downloaded `mingw-16.1-without-git.exe`.
2. Unzipped the file to `c:\MinGW` and added `c:\MinGW\bin` directory to my PATH.
3. Updated the `INCLUDES` line in the `Makefile` to read:
    ```
    INCLUDES=-Iinclude -Ipolygon -Ipotrace -Isrc -IC:\MinGW\include
    ```
4. Added the following at the top of `common/geometry/shape_poly_set.cpp`:
    ```
    #define _USE_MATH_DEFINES 
    #include <cmath>
    ```
5. Ran `make`, it compiled the project and created `bitmap2component.exe`
6. Ran `strip bitmap2component.exe` to make the file smaller (~4MB → 1.4MB, so it fits on a 💾)

**Comment by badgeek** [comment]
@urish @kasbah thanks gonna check and merge this next sunday!

**Comment by urish** [comment]
@badgeek 👍  had a chance to look at the PR?

**Comment by badgeek** [comment]
@urish @kasbah @fabiodblab released 2.0.17 thanks all!

**Comment by urish** [comment]
Thanks! I see the bitmap2component binaries haven't been updated in the repo. I published updated Windows / Linux binaries here: https://github.com/urish/svg2shenzhen/tree/enable-all-layers-bin/inkscape/svg2shenzhen

**Comment by badgeek** [comment]
@urish updated the binaries + osx built thanks!

**Pull request merged**